upgrade-notes/backend_20240717_111911.md

```diff
@@ -0,0 +1,79 @@
+#### limited item counts in sliders ([#3244](https://github.com/shopsys/shopsys/pull/3244))
```

Comment:
_nitpick_ 🤓 
the first heading should be more like "do this". Like `limit item counts in sliders`